### DISCUSSION TITLE
How to integrate OrchardCore.OpenId to an Authentication Server using Microsoft Identity

### DISCUSSION THREAD

**Discussion by maniani-dev** [OrchardCMS/OrchardCore#13519] [Q&A]
I am trying to figure out how to add OrchardCore.OpenId to an existing Authentication Server pipeline.

I thought I'd find something like AddOrchardOpenId() to add the OrchardCore OpenId module but not joy.

I am interested in seeing the OpenId endpoints without UI which AddOrchardCms() already does it perfectly out of the box

forgive me if I am barking at the wrong tree here

**Comment by ns8482e** [comment]
OpenId is core module, you can add reference to your web app to enable openid related features. Documented here https://docs.orchardcore.net/en/latest/docs/reference/modules/OpenId/
